hw09-jdbc/src/main/java/ru/otus/hw09/service/DbExecutorImp.java

```diff
@@ -0,0 +1,62 @@
+package ru.otus.hw09.service;
+
+import org.slf4j.LoggerFactory;
+import ru.otus.hw09.jdbc.JDBCTemplateImp;
+import ru.otus.hw09.jdbc.MyConnectionImp;
+
+import java.util.Objects;
+
+public class DbExecutorImp implements DbExecutor {
+
+    private static final org.slf4j.Logger logger = LoggerFactory.getLogger(DbExecutorImp.class);
+
+    private JDBCTemplateImp<Object> jdbcTemplateImp;
```

Comment:
интерфейс, а не реализация
```suggestion
    private JDBCTemplate<Object> jdbcTemplateImp;
```

Другие подобные места сами посмотрите.

Reply:
Внес изменения.